OpenRA.Game/Network/Order.cs

```diff
@@ -236,6 +236,11 @@ public static Order StartProduction(Actor subject, string item, int count)
 			return new Order("StartProduction", subject, false) { ExtraData = (uint)count, TargetString = item };
 		}
 
+		public static Order StartPriorityProduction(Actor subject, string item, int count)
```

Comment:
You can avoid adding a new order type by (ab)using the `Queued` bool on StartProduction.

Reply:
You mean setting it to `true` and then checking that to decide if the item has priority? I don't know what the purpose of `Queued` is but I guess that was not the intention for it :-) If you insist I can set it up like that -- you are more familiar with the engine after all.

Reply:
I've implemented it by using the `Queued` bool. I'm not sure if I made it properly as I'm new to C#.